src/DotVVM.Compiler/Resolving/DotvvmServiceConfiguratorResolver.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using DotVVM.Compiler.Exceptions;
+using DotVVM.Framework.Configuration;
+using DotVVM.Framework.Utils;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace DotVVM.Compiler.Resolving
+{
+    public class DotvvmServiceConfiguratorResolver : IStartupClassResolver
```

Comment:
I would rename `IStartupClassResolver` to `IServiceConfiguratorResolver`. Or is it also used to resolve `DotvvmStartup` class?